yarn-project/pxe/src/pxe_oracle_interface/pxe_oracle_interface.ts

```diff
@@ -497,16 +488,19 @@ export class PXEOracleInterface implements ExecutionDataProvider {
         // Fetch the logs for the tags and iterate over them
         const logsByTags = await this.aztecNode.getLogsByTags(tagsForTheWholeWindow);
 
-        logsByTags.forEach((logsByTag, logIndex) => {
+        logsByTags.forEach(async (logsByTag, logIndex) => {
```

Comment:
async forEach?

Reply:
Why are you looking at my unfinished code. It's like observing me dressing up in the morning

Reply:
I was benchmarking and this popped up in my notifications. Worth it just for reading that answer and spitting coffee all over my desk 🤣